src/web/components/Input/CheckboxInput.spec.tsx

```diff
@@ -2,11 +2,124 @@ import { composeStories } from '@storybook/testing-react';
 import { render, screen, waitFor, waitForElementToBeRemoved } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
+import { Form } from '../Core/Form';
+import { CheckboxInput } from './CheckboxInput';
 import * as stories from './CheckboxInput.stories';
 
-const { WithValidation } = composeStories(stories);
+const { WithValidation, OneOption } = composeStories(stories);
 
 describe('CheckboxInput', () => {
+  it('should work with one option', async () => {
```

Comment:
Similar to above, can you please expand on what "work" means? Same applies to all other tests